components/finder.rst

```diff
@@ -82,7 +82,7 @@ directory to use for the search::
 Search in several locations by chaining calls to
```

Comment:
👍 for Wouter's suggestion.

@mickaelandrieu we're including more code, with short comments to explain them, as we've found that people mostly scan to code-blocks anyways. This is a situation where this isn't a *huge* win, because the text between the paragraphs is so short, but I still like.